src/libs/runtime-utils.js

```diff
@@ -70,8 +70,9 @@ export const runtimeCost = ({ runtimeConfig, status }) => {
 }
 
 export const getGalaxyCost = app => {
-  return hourlyKubernetesAppCost(app) + persistentDiskCost({ size: 250 + 10 + 100, status: 'Running' })
-  // 250Gb for the NFS disk, 10Gb for the postgres disk, and 100Gb for the boot disk
+  // numNodes * price per node + diskCost
+  return app.kubernetesRuntimeConfig.numNodes * hourlyKubernetesAppCost(app) + persistentDiskCost({ size: 250 + 10 + 100 + 100, status: 'Running' })
```

Comment:
There is only one old Galaxy app that exists and was made by a Galaxy team person so I think it's safe to make this change